index.html

```diff
@@ -1,251 +1,505 @@
+              <a href="#"><i class="fab fa-github icons"></i></a>
+            </li>
+            <li>
+              <a href="#"><i class="fab fa-linkedin-in"></i></a>
+            </li>
+            <li>
+              <a href="#"><i class="fab fa-angellist"></i></a>
+            </li>
+            <li>
+              <a href="#"><i class="fab fa-twitter"></i></a>
+            </li>
+            <li>
+              <a href="#"><i class="fab fa-medium-m"></i></a>
+            </li>
           </ul>
         </div>
-        <button class="primary-button primary-color font-inter">See Project</button>
       </div>
-    </div>
-    <div class="grid-container">
-      <div class="grid-items d-flex">
-        <div class="grid-item-card text-white">
-          <div class="grid-item-card-content">
-            <h2 class="font-crete-round">Profesional Art Printing Data</h2>
-            <p class="font-inter">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
-            </p>
-            <div class="box-container d-flex">
-              <ul class="d-flex font-inter">
-                <li class="box">html</li>
-                <li class="box">bootstrap</li>
-                <li class="box">Ruby</li>
-              </ul>
-            </div>
+      <div class="desktop-headline-bg-section d-flex">
+        <div class="desktop-headline-bg-container">
+          <img
+            class="desktop-headline-bg"
+            src="./public/images/Illustration-header@2x.png"
+            alt="background-illustration-image"
+          />
+        </div>
+      </div>
+    </section>
+    <section class="works-section">
+      <div class="d-flex intro">
+        <h1 class="font-crete-round text-xl heading-recent-works">
+          My Recent Works
+        </h1>
+        <div class="ruler hide-on-small show-on-large"></div>
+      </div>
+      <hr />
+      <div class="multi-post-container d-flex">
+        <img
+          src="/public/images/img-placeholder-1.png"
+          alt="multi-post-image"
+        />
+        <div class="multi-post-content">
+          <h2 class="font-crete-round">Multi-Post Stories</h2>
+          <p class="font-inter">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard dummy text ever
+            since the 1500s, when an unknown printer took a standard dummy text.
+          </p>
+          <div class="box-container">
+            <ul class="d-flex font-inter list-none">
+              <li class="box">css</li>
+              <li class="box">html</li>
+              <li class="box">bootstrap</li>
+              <li class="box">Ruby</li>
+            </ul>
           </div>
-          <button class="primary-button primary-color font-inter">See Project</button>
+          <button class="primary-button primary-color font-inter">
+            See Project
+          </button>
```

Comment:
- [ ] Please ensure that the See project button redirects the user to the project live demo or your GitHub repository.

Reply:
Thanks for the feedback.  
Sincerely speaking I am confused about this change because ensuring that the button redirects to the demo on the GitHub repository are not among the list of requirements in the project instructions. 
Therefore, can you please explain clearly why I should make this change when it's listed in the requirements?